libs/version/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@ set(version_headers hpx/version.hpp)
 
 set(version_sources version.cpp)
 
-if(HPX_WITH_PARCELPORT_MPI)
+if((HPX_WITH_NETWORKING AND HPX_WITH_PARCELPORT_MPI) OR HPX_MPI_WITH_FUTURES)
+  include(HPX_SetupMPI)
```

Comment:
Apparently to fix some compilation issues (see [here](https://github.com/STEllAR-GROUP/hpx/commit/ed56387187e)) but we may add it directly to the `hpx` target as the mpi include is in `src/hpx_init.cpp`

Reply:
Ah, I meant if you know why we include `HPX_SetupMPI` and then use the macro, instead of just including `HPX_SetupMPI` (and removing the macro of course)?

Reply:
How do you suggest to proceed? Should we leave this to another PR?

Reply:
Let's leave it as it is in this PR. It's not critical to change it now.